### ISSUE TITLE
`pipe(...)` only POS tags the final item?

### ISSUE THREAD

**Issue by mbforbes** [explosion/spaCy#2215]
I've been using `pipe(...)` to process a sequence of texts in parallel, but I recently noticed that only the final document is POS tagged:

```python
import spacy
sp = spacy.load('en')
[print(p) for p in sp.pipeline]

for doc in sp.pipe(['Sentence 1.', 'Sentence 2.', 'Sentence 3.']):
    print('txt: "{}", parsed: {}, tagged: {}'.format(
        doc, doc.is_parsed, doc.is_tagged))
```

outputs:

```
('tagger', <spacy.pipeline.Tagger object at 0x7fba7f55c748>)
('parser', <spacy.pipeline.DependencyParser object at 0x7fba7f44c468>)
('ner', <spacy.pipeline.EntityRecognizer object at 0x7fba7f44c4c0>)
txt: "Sentence 1.", parsed: True, tagged: False
txt: "Sentence 2.", parsed: True, tagged: False
txt: "Sentence 3.", parsed: True, tagged: True
```

If I run this over 20 items, only the last 1 is tagged.

However, if I run this over 10,000 items, 10 get tagged (they are widely spread apart).

## Your Environment

* Operating System: Ubuntu (same result on macOS)

`python -m spacy info --markdown` on Ubuntu:

* **spaCy version:** 2.0.11
* **Platform:** Linux-4.4.0-116-generic-x86_64-with-debian-stretch-sid
* **Python version:** 3.6.2
* **Models:** en, en_core_web_lg

**Comment by ines** [comment]
Thanks for the report – I'm pretty sure this was caused by an unfortunate typo that would cause `is_tagged` to not be set correctly. We've already fixed this in

The fix is also included in the current development release – so if you want, you could test it with `v2.0.12.dev0` and see if the problem is resolved?

```bash
pip install spacy==v2.0.12.dev0
```

**Comment by mbforbes** [comment]
Phew! You're right, it is fixed in `v2.0.12.dev0`. Thank you for the prompt reply!
